GOVERNANCE.md

```diff
@@ -4,83 +4,138 @@ This document defines project governance for the Strimzi project.
-The maintainers are identified in the [`MAINTAINERS`](MAINTAINERS) file. 
+A maintainer is actively responsible for the overall development of the whole Strimzi project and all its components.
+The maintainers are identified in the [`MAINTAINERS`](MAINTAINERS) file.
 
-#### Changes in Maintainership
+Every maintainer is, in effect, also an owner for all components and does not have to be mentioned in the `COMPONENT-OWNERS` file.
 
-New maintainers are proposed by an existing maintainer and are elected by a ⅔ majority maintainers vote.
-Maintainers can be removed by a ⅔ majority maintainers vote.
+### Emeritus code owners and maintainers
 
-#### Github Project Administration
+An emeritus code owner, or an emeritus maintainer, is someone who is formerly had that role but is no longer actively involved with the project.
+Emeritus code owners and Emeritus maintainers are identified in the [`EMERITUS`](EMERITUS) file.
 
-Maintainers will be added to the collaborators list of the Strimzi repository with "Write" access.
-After 6 months a maintainer will be given "Admin" access to the Strimzi repository.
+In the rest of this document the term 'maintainer' should be taken to mean 'active maintainer' unless qualified using the term 'emeritus maintainer'.
 
-### Component owners
+#### Changes in Maintainership
 
-The component owners are responsible for development of a specific subproject or component within Strimzi.
-Such components might be represented by a separate GitHub repository within the Strimzi organization (e.g. `strimzi-kafka-bridge`) or by a subdirectory in a GitHub repository (e.g. `./documentation/` in `strimzi-kafka-operator`).
-The components owners are identified in the [`COMPONENT-OWNERS`](COMPONENT-OWNERS) file in this repository which also includes the component for which they are responsible.
+New maintainers are proposed by an existing maintainer and are elected by a vote (see later for the rules of this vote).
+If the vote succeeds the proposed maintainer will be invited to become a maintainer.
+If the proposed maintainer accepts the invitation:
+* Their name will be added to the `MAINTAINERS` file.
+* They will be added to the collaborators list of the Strimzi repository with "Write" access
+* After 6 months they will be given "Admin" access to the Strimzi repository.
 
-By definition, every maintainer is also an owner for all components and does not have to be mentioned in the `OWNERS` list.
+Maintainers can be removed by a vote.
+If the vote succeeds:
+* Their name will be removed from the `MAINTAINERS` file.
+* They will be removed to the collaborators list of the Strimzi repository
+
+Maintainers who wish to become emeritus simply mark the fact by opening a PR against this repository moving their name to the `EMERITUS` file.
+The process for reverting from an emeritus maintainer to an active maintainer is the same as for new maintainers.
 
 #### Changes in component ownership
```

Comment:
I don't think it makes sense to have this as a subchapter of `Emeritus code owners and maintainers`. It should be a the same level.